### ISSUE TITLE
hyperparameters

### ISSUE THREAD

**Issue by joemzhao** [asafamr/SymPatternWSI#2]
Hi Asaf,

Thanks for the nice work and sharing the code. I am wondering how should we pick other parameters, e.g., n-represents and  n-samples-side? I tried to go through the code but didn't find a validation set.

Thanks!

**Comment by asafamr** [comment]
Hi, they were tuned on another dataset - SemEval 2010 WSI. The settings in 2010 are somewhat different and we could not fit this into the small paper. I hope we'll upload a followup paper in few weeks or so where this is described better (also had some serious gains from adaptation to BERT's MLM). note that the task scores are quite robust to perturbations of these parameters

**Comment by joemzhao** [comment]
> Hi, they were tuned on another dataset - SemEval 2010 WSI. The settings in 2010 are somewhat different and we could not fit this into the small paper. I hope we'll upload a followup paper in few weeks or so where this is described better (also had some serious gains from adaptation to BERT's MLM). note that the task scores are quite robust to perturbations of these parameters

Hi Asaf,

Ok I see, and thanks a lot for the information! -- and yes I guess BERT would introduce improvements due to the MLM which would result in relatively good substitutes without using the symmetric patterns.
